### DISCUSSION TITLE
​

### DISCUSSION THREAD

**Discussion by dystopiandev** [mikro-orm/mikro-orm#3072] [Q&A]

**Comment by B4nan** [comment]
You will need to create new type for that, the provided one is designed to work only with SQL drivers.

**Reply by B4nan** [reply]
I really dont know what you expect here. If you need help, provide details. I never used "native uuid" in mongo, so lets start by you telling me how that works with the native mongo node driver.

**Comment by dystopiandev** [comment]


**Reply by B4nan** [reply]
Yes, that would be nice. PR welcome.

Btw I asked you something, you gave me two links, both seems to be something you just googled and didn't even try to open. First one is mongo docs - nothing node/js related, the other is a deprecated mongoose plugin - a bit better, but still. I asked how to do what you want in nodejs mongo driver - I literally want you to provide code and explanation of how to do what you want. If you can do that, you should know how to design your custom type.

From that mongoose plugin it seems they just use buffers, if so, take a look at the `BlobType`.
